src/components/CreateEnvironment/index.tsx

```diff
@@ -42,13 +43,21 @@ export default function CreateEnvironment() {
     [],
   );
 
-  const [name, setName] = useState("");
-  const [description, setDescription] = useState("");
-  const [path, setPath] = useState("");
-  const [tags, setTags] = useState<string[]>([]);
-  const [selectedPackages, setSelectedPackages] = useState<Package[]>([]);
+  const [name, setName] = useLocalStorage<string>("environments-selectedname", "");
```

Comment:
Depending on how the generics are set up with `useLocalStorage` the `<string>` here may be redundant as it'll get the type from the empty string default value.

NB: This is not true for empty string arrays, as TypeScript won't know the type of the element.